### PR TITLE
Added cuda GPU plugin

### PR DESCRIPTION
**Pull request by Yuav**

### CODE REVIEW

**Comment by Yuav** [comment]
The acceptance test for Debian timed out. What's the preferred way to trigger restart of that job?

File: manifests/plugin/cuda.pp [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# cuda plugin
```
Comment (vinzent) [inline manifests/plugin/cuda.pp]: I would start directly with puppet-strings docs : https://github.com/puppetlabs/puppet-strings#documenting-puppet-classes-and-defined-types

Comment (bastelfreak) [inline manifests/plugin/cuda.pp]: +1 for puppet-strings

Comment (Yuav) [inline manifests/plugin/cuda.pp]: Got an example?

**Comment by Yuav** [comment]
This what you're looking for?

**Comment by bastelfreak** [comment]
Thanks @Yuav!

**Pull request merged**